chart/values.yaml

```diff
@@ -92,6 +95,14 @@ db:
   jwtSecretKeyRef:
     name: incident-commander-postgrest-jwt
     key: PGRST_JWT_SECRET
+  connectionPooler:
+    # enabled: ""
```

Comment:
```suggestion
    enabled: false
```

Reply:
this would require us to enable pooler at 2 places.